docs/modules/reference/pages/daemons/daemon-config-files/statsd.adoc

```diff
@@ -0,0 +1,15 @@
+[[ref-daemon-config-files-statsd]]
+= Statsd
+
```

Comment:
Could we add an introductory sentence here?

Reply:
```suggestion

The Statsd daemon generates statistical TopN reports based on node data.

```